CHANGES

```diff
@@ -18,12 +18,18 @@ The rules for this file:
   * 0.5.0
 
 Enhancements
+  - Visualisation module will change the data according to input unit (PR
+  #129).
+  - The parser output dataframe with metadata (PR #129).
 
 Deprecations
 
 Fixes
 
 Changes
+  - alchemlyb.concat added to replace pd.concat (PR #129).
+  - postprocessors.units module for unit conversion (PR #129).
```

Comment:
Add under enhancements. (The "Changes" parts are listed separately, see other comments.)

Reference #125 (generally, the issue is more important than the PR, you can include both if you like, but definitely reference the issue)